src/cobra/summary/model_summary.py

```diff
@@ -0,0 +1,425 @@
+            ]
+
+        if "minimum" in frame.columns and "maximum" in frame.columns:
+            frame["range"] = list(
+                frame[["minimum", "maximum"]].itertuples(index=False, name=None)
+            )
+            return frame[["metabolite", "reaction", "flux", "range"]]
+        else:
+            return frame[["metabolite", "reaction", "flux"]]
+
+    @staticmethod
+    def _string_table(frame: DataFrame, float_format: str, column_width: int) -> str:
+        """
+        Create a pretty string representation of the data frame.
+
+        Parameters
+        ----------
+        frame : pandas.DataFrame
+            A table of fluxes.
+        float_format : str
+            Format string for floats.
+        column_width : int
+            The maximum column width for each row.
+
+        Returns
+        -------
+        str
+            The data frame formatted as a pretty string.
+
+        """
+        frame.columns = [header.title() for header in frame.columns]
+        return frame.to_string(
+            header=True,
+            index=False,
+            na_rep="",
+            formatters={
+                "Flux": f"{{:{float_format}}}".format,
+                "Range": lambda pair: f"[{pair[0]:{float_format}}; "
+                f"{pair[1]:{float_format}}]",
+            },
+            max_colwidth=column_width,
+        )
+
+    @staticmethod
+    def _html_table(frame: DataFrame, float_format: str) -> str:
+        """
+        Create an HTML representation of the data frame.
+
+        Parameters
+        ----------
+        frame : pandas.DataFrame
+            A table of fluxes.
+        float_format : str
+            Format string for floats.
+
+        Returns
+        -------
+        str
+            The data frame formatted as HTML.
+
+        """
+        frame.columns = [header.title() for header in frame.columns]
+        return frame.to_html(
+            header=True,
+            index=False,
+            na_rep="",
+            formatters={
+                "Flux": f"{{:{float_format}}}".format,
+                "Range": lambda pair: f"[{pair[0]:{float_format}}; "
+                f" {pair[1]:{float_format}}]",
+            },
+        )
+
+    def _string_objective(self, names: bool) -> str:
+        """
+        Return a string representation of the objective.
+
+        Parameters
+        ----------
+        names : bool, optional
+            Whether or not elements should be displayed by their common names.
+
+        Returns
+        -------
+        str
+            The objective expression and value as a string.
+
+        """
+        if names:
+            objective = " + ".join(
+                [f"{coef} {rxn.name}" for rxn, coef in self._objective.items()]
+            )
+        else:
+            objective = " + ".join(
+                [f"{coef} {rxn.id}" for rxn, coef in self._objective.items()]
+            )
+        return f"{objective} {self._objective_value}"
```

Comment:
Can we have `return f"{objective} = {self._objective_value}"`? I feel this will improve the readability.